### DISCUSSION TITLE
Are there any known issues with EIP-6963 connection in RainbowKit?

### DISCUSSION THREAD

**Discussion by himanshu-leap** [rainbow-me/rainbowkit#2221] [General]
I came across this issue - https://github.com/rainbow-me/rainbowkit/issues/2216, so wanted to confirm if there are any known issues.

**Comment by DanielSinclair** [comment]
No known issues. This seems to be a wallet-specific issue, rather than on the RainbowKit side. Leaving the GitHub Issue open to see if we collect any other reports and to notify the wallet developer.
